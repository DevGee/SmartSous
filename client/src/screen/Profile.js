import React, { Component } from 'react';
import { StyleSheet, Text, View } from 'react-native';
import { Button } from 'react-native-elements';
import axios from 'axios';

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});

class Profile extends Component {
  constructor() {
    super();
    this.state = {
      status: 'here',
      text: 'placeholder',
    };
  }
  componentDidMount() {

  }
  getTest() {

  }
  render() {
    return (
      <View style={styles.container}>
        <Text>{this.state.status}</Text>
        <Text>{this.state.text}</Text>
        <Button
          title='Logout'
         />
      </View>
    );
  }
}

export default Profile;
