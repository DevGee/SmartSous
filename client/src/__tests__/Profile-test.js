import 'react-native';
import renderer from 'react-test-renderer';
import React from 'react';
import Profile from '../screen/Profile';

test('renders correctly', () => {
  const tree = renderer.create(<Profile/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
