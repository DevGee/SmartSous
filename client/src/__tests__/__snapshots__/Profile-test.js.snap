// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
      "paddingTop": 23,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "stretch",
        "backgroundColor": "grey",
        "flex": 1,
        "justifyContent": "center",
        "paddingTop": 23,
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Array [
            Object {
              "margin": 7,
            },
            Object {
              "borderRadius": 19,
              "height": 34,
              "width": 34,
            },
            Object {
              "shadowColor": "rgba(0,0,0, .4)",
              "shadowOffset": Object {
                "height": 1,
                "width": 1,
              },
              "shadowOpacity": 1,
              "shadowRadius": 1,
            },
            Object {
              "alignItems": "center",
              "backgroundColor": "black",
              "justifyContent": "center",
            },
            Object {
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
          ],
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 15,
            },
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
            ],
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        î¡¹
      </Text>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          Object {
            "fontSize": 35,
          },
          undefined,
          undefined,
          undefined,
          Object {},
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      Full Name
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "stretch",
        "backgroundColor": "lightgrey",
        "flex": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "marginLeft": 15,
            "marginRight": 15,
          },
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "width": "80%",
            },
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#9E9E9E",
                "flexDirection": "row",
                "justifyContent": "center",
                "padding": 19,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Object {
                "padding": 12,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "white",
                    "fontSize": 20,
                  },
                  undefined,
                  Object {
                    "fontSize": 17.5,
                  },
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            Choose picture
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
